SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,90 @@
+# Supported Platforms
+
+This document outlines the platform support matrix for different versions of the Microsoft build of Go.
+
+## Support Policy
+
+The Microsoft build of Go follows the upstream Go [Release Policy](https://go.dev/doc/devel/release#policy). This means we support each major release (1.X) until there are two newer major releases. A new Go major version is [released every six months](https://github.com/golang/go/wiki/Go-Release-Cycle), so each Go major version is supported for about one year.
+
+## Currently Supported Versions
+
+As of July 2025, the following Go versions are actively supported:
+
+- **Go 1.25** (upcoming release)
```

Comment:
I don't believe we actively support releases that haven't released. 😄 

More generally, I would recommend removing essentially all "current" mentions in this doc to avoid it becoming a maintenance sink. IMO it's best to simply have a list of all actively supported platforms plus what's in `main` plus a static changelist of what version adds/drops support for specific platforms. (Like the FIPS doc.)

Reply:
agreed, updated